src/app/components/demo-components.service.ts

```diff
@@ -198,6 +198,13 @@ export class SkyDemoComponentsService {
         summary: 'The infinite scroll component dynamically loads data as users scroll.',
         url: '/components/infinite-scroll',
         getCodeFiles: () => this.getDemoFiles('SkyInfiniteScrollDemoComponent')
+      },
+			{
+        name: 'Inline form',
+        icon: 'keyboard-o',
+        summary: 'The inline form component renders a form in context rather than in a modal.',
```

Comment:
In the corresponding file, we say "in the current view" rather than "in context." I think we should be consistent. Personally, I think "in the current view is clearer. But either way, I'd use the same wording in both places.

Reply:
"current view" wording seems better to me as well.